### ISSUE TITLE
Hangs up on creating svg with Minify option selected

### ISSUE THREAD

**Issue by MusKaya** [felixfbecker/svg-screenshots#76]
Hi,

On https://medium.com/apache-pinot-developer-blog/real-time-analytics-with-presto-and-apache-pinot-part-ii-3d09ff937713,

With all of the 4 options selected (Inline resources, Keep links, Minify SVG, Pretty-print SVG) and with "Download SVG" as target, the process just hangs up and never completes.

Thanks for the awesome extension!

**Comment by felixfbecker** [comment]
Yeah not sure where it's hanging, it must be within SVGO, as the extension just calls SVGO. Try disabling "Minify" in the meantime.
